src/client/java/README.md

```diff
@@ -1,8 +1,8 @@
 ## Description
-This module is DAOS DFS Java API and DAOS DFS implementation of Hadoop FileSystem. There are two submodules,
-daos-java-api and hadoop-daos.
+This module is DAOS Java client and DAOS DFS implementation of Hadoop FileSystem. There are two submodules,
+daos-client and hadoop-daos.
 
-### daos-java-api
+### daos-client
```

Comment:
Do we really need to put "client" in there since i think this is fairly implicit? What about daos-java or java-daos?

Reply:
let me change it to daos-java. thanks.